polyfill/index.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+require('./fetch');
```

Comment:
The problem with having a single polyfill file instead of a set of options is that now everybody who supports IE11 will need to include the fetch polyfill even though they may not use it. I'd suggest that instead we either make the polyfill configurable or *not* include `fetch` — only include the things that we know users will need (e.g. Promises), will almost certainly use (e.g. Object.assign), or are necessary for Babel transforms to work correctly (have you looked into these?).

Reply:
I have removed `fetch` for now. My initial reasoning behind this idea was that our litmus test for modern browser tests for `fetch` and `Object.assign`, any browser which passes this would not have to load any of the polyfill code. I think I will open a new ticket to discuss this.